openmc/surface.py

```diff
@@ -558,6 +723,37 @@ def z0(self, z0):
         check_type('z0 coefficient', z0, Real)
         self._coeffs['z0'] = z0
 
+    def bounding_box(self, side):
+        """Determine an axis-aligned bounding box.
+
+        An axis-aligned bounding box for surface half-spaces is represented by
+        its lower-left and upper-right coordinates. If the half-space is
+        unbounded in a particular direction, numpy.inf is used to represent
+        infinity.
+
+        Parameters
+        ----------
+        side : {'+', '-'}
+            Indicates the negative or positive half-space
+
+        Returns
+        -------
+        numpy.array
+            Lower-left coordinates of the axis-aligned bounding box for the
+            desired half-space
+        numpy.array
+            Upper-right coordinates of the axis-aligned bounding box for the
+            desired half-space
+
+        """
+
+        if side == '-':
+            return (np.array([self.x0 - self.r, -np.inf, self.z0 - self.r]),
+                    np.array([self.x0 + self.r, np.inf, self.y0 + self.r]))
```

Comment:
ditto, should the final `self.y0 + self.r` instead be `self.z0 + self.r`? I'm probably not thinking this through very clearly at this time of the day, so correct me if I'm wrong.